bindings/gumjs/gumduksymbol.c

```diff
@@ -182,6 +184,22 @@ GUMJS_DEFINE_FUNCTION (gumjs_symbol_find_functions_matching)
   return 1;
 }
 
+GUMJS_DEFINE_FUNCTION(gumjs_load_module)
+{
+	GumDukScope scope = GUM_DUK_SCOPE_INIT(args->core);
```

Comment:
```suggestion
  GumDukScope scope = GUM_DUK_SCOPE_INIT (args->core);
```

- Indentation should be 2 spaces, no tabs.
- Always space before the opening parenthesis in function calls and macro invocations.
(Here and elsewhere.)

Style guide [here](https://github.com/frida/frida/blob/master/CONTRIBUTING.md).